pkg/downloader/manager.go

```diff
@@ -823,21 +825,40 @@ func (m *Manager) loadChartRepositories() (map[string]*repo.ChartRepository, err
 		return indices, errors.Wrapf(err, "failed to load %s", m.RepositoryConfig)
 	}
 
+	var (
+		g  errgroup.Group
+		mu sync.Mutex
+	)
 	for _, re := range rf.Repositories {
-		lname := re.Name
-		idxFile := filepath.Join(m.RepositoryCache, helmpath.CacheIndexFile(lname))
-		index, err := repo.LoadIndexFile(idxFile)
-		if err != nil {
-			return indices, err
-		}
+		re := re
```

Comment:
nit pick: Its not necessary as of go 1.22

Reply:
removed the line